### PR TITLE
APEXMALHAR-2064 moved WindowDataManager to org.apache.apex.malhar.lib…

### PR DESCRIPTION
**Pull request by otterc**
….wal

### CODE REVIEW

File: apps/pom.xml [hunk 1]
```diff
@@ -168,6 +168,13 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
```
Comment (tweise) [inline apps/pom.xml:171]: Should be enabled through parent.

**Comment by tweise** [comment]
The diff shows unrelated changes. Can you please rebase.

File: apps/pom.xml (hunk 1 above)
Comment (chandnisingh) [inline apps/pom.xml]: apps is not a child module of the main module

Comment (otterc) [inline apps/pom.xml:171]: Ok I see.

**Pull request closed without merging**